### PR TITLE
Make scripts compatible with python2+3

### PR DESCRIPTION
**Pull request by bsaund**
Mostly fixing print statements.

`manual_motion` and `arm_command_gui` runs on noetic + python3 and kinetic +python2 (realtime. I was not able to fully test the gui)

### CODE REVIEW

File: victor_hardware_interface/scripts/arm_command_gui.py [hunk 1]
```diff
@@ -185,22 +185,22 @@ def __init__(self, parent_layout, arm_name, box_row):
...
         while (not self.arm_status_updated) or (not self.gripper_status_updated):
             time.sleep(0.01)
-        print 'Done'
+        print('Done')
 
         self.reset_arm_slider_to_current_value()
         self.reset_gripper_position_slider_to_current_value()
 
-        print 'Getting', self.name, 'current control mode ...',
+        print('Getting {} current control mode ...'.format(self.name))
```
Comment (PeterMitrano) [inline victor_hardware_interface/scripts/arm_command_gui.py:197]: I'd use f-strings

    print(f'Getting {self.name} current control mode')

File: victor_hardware_interface/scripts/arm_command_gui.py [hunk 2]
```diff
@@ -385,30 +385,30 @@ def joint_slider_moved(self, position, joint_name):
 
     def set_stiffness(self, stiffness):
         self.stiffness = self.stiffness_map[stiffness]
-        print self.stiffness
+        print("Stiffness: ".format(self.stiffness))
```
Comment (PeterMitrano) [inline victor_hardware_interface/scripts/arm_command_gui.py:388]: f string

File: victor_hardware_interface/scripts/manual_motion.py [hunk 3]
```diff
@@ -56,8 +56,8 @@ def run_lowpass(self, msg):
         self.prev_time = now
 
         if dt > 1:
-            print "Incredibly long time between messages. Something is weird"
-            print "dt: ", dt
+            print("Incredibly long time between messages. Something is weird")
+            print("dt: {}".format(dt))
```
Comment (PeterMitrano) [inline victor_hardware_interface/scripts/manual_motion.py:60]: f string

**Review by PeterMitrano — APPROVED** [review]
LGTM

File: victor_hardware_interface/scripts/arm_command_gui.py (hunk 1 above)
Comment (bsaund) [reply victor_hardware_interface/scripts/arm_command_gui.py:197]: How do you get f-strings to work in python2?

Comment (PeterMitrano) [reply victor_hardware_interface/scripts/arm_command_gui.py:197]: oh fair enough, you probably can't. Nevermind then.

**Pull request merged**